### ISSUE TITLE
"volume out of range" error confusing, doesn't specify RemoteTech

### ISSUE THREAD

**Issue by Gaiiden** [KSP-KOS/KOS#1580]
I had my script show me this error:

>Volume is out of range
>(Cannot Show kOS Error Location - error might really be internal. See kOS devs.)

Which originally had me thinking it was some internal issue thanks to the second line. Took me a few tries at debugging to realize it was because I had lost my RemoteTech signal over the horizon while I was logging data to the Archive volume. I think if only the first line had popped up alone I still would have been slightly confused at first. Would be better if stated:

"Connection to KSC lost, volume is no longer accessible"

**Comment by hvacengi** [comment]
The `(Cannot Show kOS Error Location - error might really be internal. See kOS devs.)` portion is related to the stack trace, rather than the error itself, which is probably more misleading than the message itself.

If the message was:
> Volume is out of range
> Error Trace:
>    (Cannot Show kOS Error Location - error might really be internal. See kOS devs.)

Would that have been more clear?

We can also update the exception text, but with the future communication update from Squad (not in version 1.1) we may be updating the text further.  My ideal message would actually be "Volume is out of communication range." because it is short, while still giving you the information that it is communication related.

**Comment by Dunbaratu** [comment]
Maybe the message about the stack trace can be changed a bit.  Basically it comes up whenever the problem can't be traced to a line of your script so we can't show you where it came from.  It *assumes* this must be because it's an internal error that's not your fault, but maybe that's not always true.

**Comment by Gaiiden** [comment]
I agree it would be better off being more generic in saying "communication", good call
